website/docs/d/key_vault_secret_versions.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Key Vault"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_key_vault_secret_versions"
+description: |-
+  Get a list of versions for an existing Key Vault Secret.
+---
+
+# Data Source: azurerm_key_vault_secret_versions
+
+Use this data source to access information about an existing Key Vault Secret's versions. The secret version values is not included. The `key_vault_secret` data source can be used to retrieve the value of a given secret version using it's `id`.
```

Comment:
With the introduction of ephemeral resources we might want to point users towards the ephemeral resource instead of the data source
```suggestion
Use this data source to access information about an existing Key Vault Secret's versions. The secret version values are not included. The `azurerm_key_vault_secret` ephemeral resource can be used to retrieve the value of a given secret version using it's `id` without storing the information in state. Alternatively the `azurerm_key_vault_secret` data source can be used to retrieve this information, but will store this information in state.
```